app/src/androidMain/kotlin/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessBusiness.kt

```diff
@@ -23,6 +23,7 @@ class AddWheelchairAccessBusiness : OsmFilterQuestType<WheelchairAccess>(), Andr
             or amenity = recycling and recycling_type = centre
             or amenity = social_facility and social_facility ~ food_bank|clothing_bank|soup_kitchen|dairy_kitchen
             or tourism = information and information = office
+            or tourism = wilderness_hut
```

Comment:
if it is added both to name & wheelchair - then it should be added in their shared section, see `// name & wheelchair` comment in line 51